### ISSUE TITLE
Better name for Wallabag option "Mark as finished instead of deleting"

### ISSUE THREAD

**Issue by Gullumluvl** [koreader/koreader#11687]
Hi all,

KOreader is really awesome, and especially the ability to synchronize with Wallabag.

I am just raising a minor potential issue here, to document the setting, and to ask if other users got confused as well: Wallabag's default action for read articles is to keep them as archive, and I had been looking for a way to have this behavior in KOreader, which by default *completely deletes articles* instead.

I only found out now that this option is named "Mark as finished instead of deleting" in the settings menu:

https://github.com/koreader/koreader/blob/715f5aa7471c87927845c17888551c0d300c731c/plugins/wallabag.koplugin/main.lua#L283

and is saved in the variable `is_archiving_deleted`.

So everything works great in reality, but the setting name is confusing in my opinion because it seems to imply that *local* articles will be marked as finished in KOreader, as per the terminology used in KOreader. It's not really possible to know because this feature is not documented in the wiki.

I would argue that its name should be changed in favor of: "Archive instead of deleting", to match the terms used in the Wallabag interface. This would need to be replicated in all translations (I use French). I just need to confirm that there is no such thing as "archive" on KOreader side, is that right? I would also argue that this option should be enabled by default to match Wallabag behavior.

I am willing to update the wiki and prepare a pull request if needed :)

Best,

**Comment by Gullumluvl** [comment]
> > I would also argue that this option should be enabled by default to match Wallabag behavior.
> 
> But it is? I think there must be some other missing factor involved.

You are right sorry, it has been enabled by default in commit 08d6fbc (Apr 19, 2022) but I use an older version.
The old defaults are still shown in the wiki page https://github.com/koreader/koreader/wiki/Wallabag.

> I seem to be somewhat responsible for this phrasing: [#5148 (comment)](https://github.com/koreader/koreader/pull/5148#discussion_r307992114)

Oh so I checked Wallabag's interface, the articles are either in the "Unread" list, or in the "Archive" list, corresponding to the API flag `archive=0` or `archive=1`, BUT the button to change it (per article) is named "Toggle mark as read". So I guess it was just a matter of personal interpretation... In case we still want to clear out any confusion, how about "Remotely mark as read instead of deleting"?

**Comment by Frenzie** [comment]
Yes, with "mark as read" I believe I was referring to Wallabag. In KOReader you mark something as finished, which is presumably how the rephrasing happened.

> In case we still want to clear out any confusion, how about "Remotely mark as read instead of deleting"?

Sounds good to me.

**Comment by Commodore64user** [comment]
> In KOReader you mark something as finished,

is there a reason for why we use "finished" instead of the more obvious and wildly used term "read"?

**Comment by Frenzie** [comment]
Because the word "read" is much harder to disambiguate in these kinds of situations. `Read` could also mean `Open` as opposed to "Readed".

<img src=https://github.com/koreader/koreader/assets/202757/eb0bff83-19cc-4cd8-bf5c-db60d7854805 width=50%>

Kobo made the same choice:

https://help.kobo.com/hc/en-us/articles/360017481154-Mark-a-book-as-Finished-on-your-Kobo-eReader

**Comment by Frenzie** [comment]
As for your follow-up question, we thought `Finished` sounded better than `Complete`.

**Comment by Commodore64user** [comment]
> Because the word "read" is much harder to disambiguate in these kinds of situations. `Read` could also mean `Open` as opposed to "Readed".

not suggesting anything here but, but `read` (past tense) means read (done reading), and `reading` means open. Not ambiguous at all.

**Comment by Frenzie** [comment]
There's nothing ambiguous about it if you've already disambiguated it is a meaningless statement.

**Comment by offset-torque** [comment]
> @Commodore64user > is there a reason for why we use "finished" instead of the more obvious and wildly used term "read"?

If you count academic articles, **Finished** fits better for this action. When reading these articles, generally people have a look at the **Methods** section and then jump to the **Results/Discussion**. Nobody reads a paper completely. So when I process a paper, I am **"Finished"** with it, but only around 10% is **"Read"**.

**Comment by Frenzie** [comment]
@Commodore64user On a related note, a little something to maybe keep in mind: somewhere you had proposed to replace neutral `title` by `book title`. As a rule of thumb I'd say shorter and less wordy is better when possible, but more important it could be an academic paper, it could be a short story/novella, it could be a newspaper article, etc.

**Comment by Commodore64user** [comment]
I am sorry to disappoint but book title, was already in use:

https://github.com/koreader/koreader/blob/caea0e8fb2e07c27670137e3ebbb4945f34b5ce0/frontend/apps/reader/modules/readerfooter.lua#L984

I just embraced it.

**Comment by Frenzie** [comment]
Sure, but embrace it the other direction. ;-)

**Comment by harmtemolder** [comment]
I'm working on a PR for #12712 and am thinking to invert the _Mark as finished instead of deleting_ setting (default = on) to _Delete instead of marking as finished_ (default = off). I'll include functionality that migrates the old setting to the new one, so that nothing is lost for existing users

**Comment by harmtemolder** [comment]
@Gullumluvl Could you help me test my PR at #12949 ?
